### PR TITLE
Improve assertion and ignore cached PHPUnit file

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- Improve some PHPUnit assertions.
- Let the `.phpunit.result.cache` file be on `.gitignore` file and this is generated by `PHPUnit 9.x.` version.

### CODE REVIEW

File: tests/DebugBar/Tests/DataCollector/AggregatedCollectorTest.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ public function testAddCollector()
         $this->c->addCollector($c = new MockCollector());
         $this->assertContains($c, $this->c->getCollectors());
         $this->assertEquals($c, $this->c['mock']);
-        $this->assertTrue(isset($this->c['mock']));
+        $this->assertArrayHasKey('mock', $this->c);
```
Comment (peter279k) [inline tests/DebugBar/Tests/DataCollector/AggregatedCollectorTest.php:20]: Using the `assertArrayHasKey` to assert the expected key is on the result array.

File: tests/DebugBar/Tests/DataCollector/TimeDataCollectorTest.php [hunk 2]
```diff
@@ -33,16 +33,16 @@ public function testStartStopMeasure()
         $this->assertEquals('bar', $m[0]['label']);
         $this->assertEquals('baz', $m[0]['collector']);
         $this->assertEquals(array('bar' => 'baz'), $m[0]['params']);
-        $this->assertTrue($m[0]['start'] < $m[0]['end']);
+        $this->assertLessThan($m[0]['end'], $m[0]['start']);
```
Comment (peter279k) [inline tests/DebugBar/Tests/DataCollector/TimeDataCollectorTest.php:36]: Using the `assertLessThan` to assert expected and result comparison.

File: tests/DebugBar/Tests/DebugBarTest.php [hunk 3]
```diff
@@ -96,7 +96,7 @@ public function testStackedData()
         $data = $this->debugbar->getStackedData();
         $this->assertArrayNotHasKey($ns, $http->session);
         $this->assertArrayHasKey($id, $data);
-        $this->assertEquals(1, count($data));
+        $this->assertCount(1, $data);
```
Comment (peter279k) [inline tests/DebugBar/Tests/DebugBarTest.php:99]: Using the `assertCount` to assert expected is same as result count.

File: tests/DebugBar/Tests/DebugBarTestCase.php [hunk 4]
```diff
@@ -17,13 +17,13 @@ public function setUp(): void
     public function assertJsonIsArray($json)
     {
         $data = json_decode($json);
-        $this->assertTrue(is_array($data));
+        $this->assertIsArray($data);
```
Comment (peter279k) [inline tests/DebugBar/Tests/DebugBarTestCase.php:20]: Using the `assertIsArray` to assert expected type is `array` type.

File: tests/DebugBar/Tests/DebugBarTestCase.php [hunk 5]
```diff
@@ -17,13 +17,13 @@ public function setUp(): void
...
     }
 
     public function assertJsonIsObject($json)
     {
         $data = json_decode($json);
-        $this->assertTrue(is_object($data));
+        $this->assertIsObject($data);
```
Comment (peter279k) [inline tests/DebugBar/Tests/DebugBarTestCase.php:26]: Using the `assertIsObject` to assert expected type is `object`.

**Review by parallels999 — APPROVED** [review]

**Pull request merged**